shaketune/motor_res_filter.py

```diff
@@ -0,0 +1,142 @@
+
+            # Ignore the motor resonance filters for smoothers from DangerKlipper
+            if shaper_type.startswith('smooth_'):
+                ConsoleOutput.print(
+                    (
+                        f'Warning: {shaper_type} type shaper on {axis} axis is a smoother from DangerKlipper '
+                        'Bleeding-Edge that already filters the motor resonance frequency range. Shake&Tune '
+                        'motor resonance filters will be ignored for this axis...'
+                    )
+                )
+                continue
+
+            # Ignore the motor resonance filters for custom shapers as users can set their own A&T values
+            if shaper_type == 'custom':
+                ConsoleOutput.print(
+                    (
+                        f'Warning: custom type shaper on {axis} axis is a manually crafted filter. So you have '
+                        'already set custom A&T values for this axis and you should be able to convolve the motor '
+                        'resonance frequency range to this custom shaper. Shake&Tune motor resonance filters will '
+                        'be ignored for this axis...'
+                    )
+                )
+                continue
+
+            # At the moment, when running stock Klipper, only ZV type shapers are supported to get combined with
+            # the motor resonance filters. This is due to the size of the pulse train that is too small and is not
+            # allowing the convolved shapers to be applied. This unless this PR is merged: https://github.com/Klipper3d/klipper/pull/6460
+            if not self._in_danger and shaper_type != 'zv':
+                ConsoleOutput.print(
+                    (
+                        f'Error: the {axis} axis is not a ZV type shaper. Shake&Tune motor resonance filters '
+                        'will be ignored for this axis... This is due to the size of the pulse train being too '
+                        'small and not allowing the convolved shapers to be applied... unless this PR is '
+                        'merged: https://github.com/Klipper3d/klipper/pull/6460'
+                    )
+                )
+                continue
+
+            # Get the current shaper parameters and store them for later restoration
+            _, axis_shaper_A, axis_shaper_T = shaper.get_shaper()
+            self._original_shapers[axis] = (axis_shaper_A, axis_shaper_T)
+
+            # Creating the new combined shapers that contains the motor resonance filters
+            if axis in {'x', 'y'}:
+                if self._in_danger:
+                    # In DangerKlipper, the pulse train is large enough to allow the
+                    # convolution of any shapers in order to craft the new combined shapers
+                    # so we can use the MZV shaper (that looks to be the best for this purpose)
+                    df = math.sqrt(1.0 - self.damping_x**2)
```

Comment:
You only use damping_x here for both x and y (while technically they should be the same it doesnt make sense to have them seperate and then just ignore the value for y)